content/en/developers/guide/unified-tagging-advanced-usage.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Unified Tagging Advanced Usage Guide
+kind: guide
+aliases:
+  - /developers/guide/unified-tagging-advanced-usage
+---
+
+## Overview
+
+This guide suggests ways to configure and migrate to [unified service tagging][1] based on specific use cases.
+
+## Using Custom Tags
+
+You can continue to use the `env` `service` and `version` tags as they are configured for unified service tagging. However, if you'd like to achieve a unified tagging experience with your own custom tags, there are some options available to you:
+
+**Note**: While some products support arbitrary tags, others have more specific expectations. As such, navigation between products might be difficult if one data source has a tag that another does not have or does not support.
+
+### Containerized Environments
+
+#### Metrics
+
+Since tags can be appended to data points, you have a lot of freedom in setting desired tags. Autodiscovered tags are automatically added to all metrics collected.
+
+#### APM
+
+`env` and `service` are core tags in APM, so it is not possible to replace them with differently named tags. However, APM does allow for data to be [aggregated along more primary tags] than just `env`. Since host tags are added to traces and trace metrics, they can be used as well (e.g., availability-zone).
+
+Trace metrics also have a specific [whitelist of tags][2]. Container tags are not included.
```

Comment:
Maybe we can move the bit about container tags to the next sentence? Or combine these two sentences? 

Something like:
> Autodiscovered tags associated with containers are added to `container_info` in span metadata.
> However, these container tags are not part of the whitelisted tags for trace metrics.